introduction_to_amazon_algorithms/jumpStart_tabular_classification/Amazon_JumpStart_Tabular_Classification_LightGBM_CatBoost.ipynb

```diff
@@ -0,0 +1,597 @@
+{
```

Comment:
<p>Branding: change all occurrences of <code>Sagemaker</code> to <code>SageMaker</code>	(fix all 4 occurrences across the 4 notebooks in this PR)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Ditto this comment. Once this is corrected, we will begin the process of merging.</p>